worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageProcessorImpl.java

```diff
@@ -136,11 +141,11 @@ public boolean startJob(CoverageMapping mapping) {
                 return false;
             }
 
-            log.info("starting search for {} during {}-{}", mapping.getContract().getContractNumber(),
+            log.info("starting search for {} during {}-{}", mapping.getContractNumber(),
                     mapping.getPeriod().getMonth(), mapping.getPeriod().getYear());
 
             // Currently, we are using the STU3 version to get patient mappings
-            CoverageMappingCallable callable = new CoverageMappingCallable(STU3, mapping, bfdClient);
+            CoverageMappingCallable callable = new CoverageMappingCallable(STU3, mapping, bfdClient, contractService);
```

Comment:
Why not get the DTO here and pass it in directly or even just pass in the the contract type from the DTO only?

Reply:
The question becomes where the DTO data comes from. contractService?